runhouse/resources/hardware/cluster.py

```diff
@@ -190,6 +197,21 @@ def default_env(self, env):
                 "Run `cluster.restart_server()` to restart the Runhouse server on the new default env."
             )
 
+    @property
+    def observability_disabled(self) -> bool:
+        return self._disable_observability
+
+    @observability_disabled.setter
```

Comment:
shouldn't these be a property instead of a setter? meaning when we call `self.observability_disabled`, thats when we compare the value passed in (self._observablity_disabled) to the value stored in the config